requirements/edx/stanford.txt

```diff
@@ -0,0 +1,10 @@
+xblock-image-modal >=0.2.0, <1.0.0
+-e git+https://github.com/openlearninginitiative/xblock-image-coding.git@cee23e3d59943066159ca13839c82a12d9ccf749#egg=xblock_image_coding
+-e git+https://github.com/Stanford-Online/xblock-qualtrics-survey@653400485196f8bb05b859c9d89de378a22e0503#egg=xblock-qualtrics-survey
+-e git+https://github.com/Stanford-Online/xblock-free-text-response@01cb026337f4fdd97e89b017142be06506e56afa#egg=xblock-free-text-response
+-e git+https://github.com/Stanford-Online/xblock-grademe.git@v0.1.0#egg=grademebutton
+-e git+https://github.com/edx-solutions/xblock-ooyala.git@32d52edaa820dbdbf846d8a84f6bccbf0b9c8218#egg=xblock_ooyala_player-master
```

Comment:
> Can we name the file "xblocks_peculiar_to_stanford.txt" then? That'll make it very clear what it is.

But this isn't just for XBlocks; it's for any kind of Python package. At that point, we don't gain any clarity by mentioning `packages_peculiar_to_stanford` because _everything_ in the requirements directory is a list of packages for something..

Don't get me wrong.. I'm all in favor of clear naming conventions, but without sounding too obtuse, I'm not sure how much more clearly we can communicate "requirements for Stanford" better than a file named `stanford` in the requirements directory..

If you can think of something more apt, I'm open to it.

Thoughts @kluo @caesar2164 ?